tensilelite/Tensile/Source/client/include/DataInitialization.hpp

```diff
@@ -517,6 +525,18 @@ namespace Tensile
                 case InitMode::TrigCos:
                 case InitMode::TrigAbsSin:
                 case InitMode::TrigAbsCos:
+                case InitMode::TrigIndSin:
```

Comment:
case InitMode::Free: ~ case InitMode::TrigAbsCos:  would return runtime error originally.

Reply:
Can we just support TrigSin ~ TrigAbsCos in getValue() without adding TrigIndXXX?

Reply:
TrigXXX we need to pass tensor sizes to get MNK
TrigIndXXX we need to pass total elements.